ui/src/entities/Team.ts

```diff
@@ -1,28 +1,48 @@
 import client from 'src/lib/client';
+import resolveStorageFile from 'src/utils/resolveStorageFile';
+import { Project } from './Project';
 
-export interface Team {
+export interface TeamResponse {
```

Comment:
Let's rename it back to `Team`. `Team` is a domain object and should be represented outside the context of a request-response cycle.